docs/_rpc/ns-admin.md

```diff
@@ -51,6 +63,29 @@ currently uses to store all its databases.
 "/home/john/.ethereum"
 ```
 
+### admin_exportChain
+
+Exports the current blockchain into a local file, the path to which is passed as first argument.
+It optionally takes a first last block number, in which case it exports only that range of blocks.
+
+It returns a boolean indicating whether the operation succeeded.
+
+| Client  | Method invocation                                                     |
+|:--------|---------------------------------------------------------------------- |
+| Console | `admin.exportChain(file, first, last)`                                |
+| RPC     | `{"method": "admin_exportChain", "params": [string, uint64, uint64]}` |
+
+### admin_importChain
+
+Imports a blockchain from a local file.
```

Comment:
blockchain -> exported list of blocks? It's a bit more correct. Also would be nice to mention that it tries to process them, so you need the state available for the parent.